core/src/main/java/org/elasticsearch/action/search/SearchRequestBuilder.java

```diff
@@ -293,6 +293,18 @@ public SearchRequestBuilder addDocValueField(String name) {
     }
 
     /**
+     * Adds a docvalue based field to load and return. The field does not have to be stored,
+     * but its recommended to use non analyzed fields.
+     *
+     * @param name The field to get from the docvalue
+     * @param format How to format the field, {@code null} to use defaults.
+     */
+    public SearchRequestBuilder addDocValueField(String name, String format) {
```

Comment:
Since we're usually OK with making breaking changes to the Transport API in a minor release it is probably ok to add the format parameter to this method if you think it makes sense.